cloudcmd/rbac.go

```diff
@@ -20,9 +22,9 @@ type CloudRbac struct {
 	Config         internal.UserConfig
 }
 
-func NewCloudRbac(MinAllowedUser Permission, config internal.UserConfig) CloudRbac {
+func NewCloudRbac(minAllowedUser Permission, config internal.UserConfig) CloudRbac {
```

Comment:
Nit: Minor comment, RBAC is acronym like API,ID... (Role Based Access Control), as I saw that you changed to upper case for acronyms.